README.md

```diff
@@ -64,38 +69,62 @@ Then this certificate must be signed by an administrator who also assigns you a
+* Clone your fork locally: `git clone https://github.com/<your-github-username>/openvpn && cd openvpn`
+* Build EASYVPN binary by running one of the following commands depending on your OS:
   * `make init_osx`
   * `make init_linux`
   * `make init_windows` then copy utils/easyvpn/easyvpn.exe at the root of this repository
-* Generate your private key and certificate request: `./easyvpn request <your username>`
-  Your private key will be generated in `cert/pki/private`, this key **must** remain **secret**.
-* Create a new Pull Request on jenkinsinfra/openvpn, staging branch: [How to Create a pull request](https://help.github.com/articles/creating-a-pull-request/)
+* Generate your private key and certificate request: `./easyvpn request <your-jenkins-username>`
+  Your private key will be generated in `./cert/pki/private`, this key **must** remain **secret**.
+* Create a new Pull Request on [jenkinsinfra/openvpn](https://github.com/jenkins-infra/openvpn), `staging` branch: [How to Create a pull request](https://help.github.com/articles/creating-a-pull-request/)
 * Open an INFRA ticket on [JIRA](https://issues.jenkins-ci.org) referencing your PR
-* Grab a cup of coffee and wait patiently for an administrator to sign your certificate request.
-* Once an admin notify you that everything is right, you can then retrieve your certificate from `./cert/pki/issued/<your_username>.crt`
+* Grab a cup of coffee and wait patiently for an administrator to sign your certificate request
+* Once an admin notify you that everything is right, you can [sync your fork](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) then pull it to retrieve your certificate from `./cert/pki/issued/<your-jenkins-username>.crt`
+* We recommend you to move the `./cert` folder to an hidden folder in your home (`~/.cert`)
+* You can finally create the config file used by your VPN client (example here for Tunnelblick, an OSX VPN client, opening this file from the Finder should launch it)  
+
+_jenkins-infra.ovpn_
+```text
+client
+remote vpn.jenkins.io 443
+ca "~/.cert/pki/ca.crt"
+cert "~/.cert/pki/issued/<your-jenkins-username>.crt"
+key "~/.cert/pki/private/<your-jenkins-username>.key"
+auth-user-pass
+dev tun
+proto tcp
+nobind
+auth-nocache
+script-security 2
+persist-key
+persist-tun
+user nobody
+group nobody
+```
 
 #### HowTo show request information
 
 * Enter in the VPN network directory: `cd cert`
-* Run `make show-req name=<username>`
+* Run `make show-req name=<your-jenkins-username>`
 
 #### HowTo show certificate information
 
-* Enter in the VPN network directory: `cd cert`
-* Run `make show-cert name=<username>`
+* Install [sops](https://github.com/mozilla/sops)
+* Enter in the VPN network directory: `cd ~/.cert`
+* Run `make decrypt`
+* Run `make show-cert name=<your-jenkins-username>`
```

Comment:
I realize, while reviewing, that this sub-section should be moved under the "Administrator" sub section, because the command `make decrypt` requires the user to be allowed to decrypt, which is only for admins...

WDYT about this? That would avoid polluting the mind of the reader who only want to get a VPN access)

Reply:
I've reorganized (and simplified some of) the first sections.

Reply:
good work!